webapp/scripts/functions.js

```diff
@@ -93,12 +100,16 @@ module.exports = {
    * @param {string} column
    */
   getUnivar(mapset, map, column) {
-    const rawArray = grass(mapset, `v.db.univar -e -g map=${map} column=${column}`).trim().split('\n')
-    return rawArray.reduce((dict, line) => {
-      const a = line.split('=')
-      dict[a[0]] = a[1]
-      return dict
-    }, {})
+    try {
+      const rawArray = grass(mapset, `v.db.univar -e -g map=${map} column=${column}`).trim().split('\n')
+      return rawArray.reduce((dict, line) => {
+        const a = line.split('=')
+        dict[a[0]] = a[1]
+        return dict
+      }, {})
+    } catch (e) {
+      console.log('v.db.univar error: ', e)
```

Comment:
Again, why hide the error from the user?

Reply:
v.db.univar returns error on some columns, but the reason for the error is obscure and is likely a data issue. Instead of letting this error stop the whole process, I believe it is better to hide those problematic columns.

Reply:
What does the error say?
And while I agree this issue shouldn't break the process, the user might wonder why that layer doesn't show up. We should ideally display a warning (not an error) so the user is informed that something has gone wrong.